### DISCUSSION TITLE
Request for samples

### DISCUSSION THREAD

**Discussion by flcl42** [NethermindEth/dotnet-libp2p#60] [Ideas]
The project contains few examples, let's add more and implement features needed for them to work well. 

Make a post or vote for a proposal!

**Comment by flcl42** [comment]
Connect to a node/list of nodes and report addresses, basic transport, agent and protocols supported by it.

**Comment by flcl42** [comment]
Query Ethereum blobs and show stats, readable content if detected

**Comment by dhuseby** [comment]
An implementation of the [universal connectivity peer](https://github.com/libp2p/universal-connectivity)
